src/modules/ecommerceTracking/populateActionField.js

```diff
@@ -0,0 +1,22 @@
+import{extractMetaTagContent} from '../extractMetaTagContent';
+
+let actionField = {
+    id : extractMetaTagContent('WT\\.tx_id'),
+    affiliation : extractMetaTagContent('WT\\.si_n'),
+    revenue : extractMetaTagContent('WT\\.tx_total'),
+    tax : '0',
+    shipping : '0'
+};
+
+export let populateActionField = (step) => {
```

Comment:
Hi @JamesWChan ,
Could add some tests to check the property and default values;

Reply:
How far have got with this?